src/views/LoginView.vue

```diff
@@ -35,60 +32,100 @@ userStore.reset();
 </template>
 
 <style>
-main {
+main#login-view {
```

Comment:
Avoid using ID for style, could create issue with specificity.
Prefer class usage